Automattic/wp-calypso
# Offer Reset: add upgrade nudge for Jetpack Security Daily (and maybe others in the future)

#### Changes proposed in this Pull Request

* Display upgrade nudge to users that have Jetpack Security Daily.
* After dismissed, don't show the nudge again.

#### Testing instructions

* Run this PR.
* Enable the `offerResetFlow` AB experiment.
* Go to the new plans section.
* Verify the upgrade nudge that appears in inside the Jetpack Security card looks as per design.

**Note**: if you dismissed the nudge and want to see it again, you can always revert the UI to its previous state by removing the stored preference. See the screenshot below.
![image](https://user-images.githubusercontent.com/3418513/89833020-b20af180-db36-11ea-95dd-8a642f11d5e8.png)

Fixes 1169247016322522-as-1188038659882088/f

#### Demo
##### Desktop
<img src="https://user-images.githubusercontent.com/3418513/89942533-2014ee80-dbf3-11ea-8490-e46e56d1d597.png" width="800" alt="Desktop version">

##### Mobile
<img src="https://user-images.githubusercontent.com/3418513/89942668-59e5f500-dbf3-11ea-9d77-32cddd729b66.png" height="800" alt="Mobile version">
